src/components/markdown-help.less

```diff
@@ -0,0 +1,92 @@
+.MarkdownHelp {
+  font-size: 18px;
+  display: grid;
+  align-content: center;
+  justify-content: center;
+  cursor: pointer;
+
+  &.is-disabled {
+    cursor: not-allowed;
+  }
+
+  &.MarkdownHelp--small {
+    font-size: 12px;
+  }
+
+  &.MarkdownHelp--block {
+    padding: 4px 12px;
+    border-radius: 5px;
+    background-color: @edu-secondary-color-light;
+    opacity: 0.25;
```

Comment:
I guess this opacity could be extracted in a semantically meaningful variable, something like `@edu-markdown-[inactive/unfocused]-opacity` 🤔

Reply:
Good idea, as it is used in two places. Done!